evalml/automl/engine/dask_engine.py

```diff
@@ -0,0 +1,83 @@
+import joblib
+from dask.distributed import Client
+
+from evalml.automl.engine.engine_base import (
+    EngineBase,
+    EngineComputation,
+    evaluate_pipeline,
+    score_pipeline,
+    train_pipeline
+)
+
+
+class DaskComputation(EngineComputation):
+    """A Future-like wrapper around jobs created by the DaskEngine."""
+
+    def __init__(self, dask_future):
+        self.work = dask_future
+        self.meta_data = {}
+
+    def done(self) -> bool:
```

Comment:
Feels a little random to do type-hinting on certain functions, but not others. Not a big deal, but just my thoughts

Reply:
I'll get rid of them! I added them at first when I was sketching stuff out and forgot every place hehe